### PR TITLE
Improve firewall module

### PR DESCRIPTION
**Pull request by juliocc**
- Simplified code structure to a single resource block for custom rules
- Added new output with all the google_compute_firewall resources
- Allow using named ranges in custom firewall rules

### CODE REVIEW

File: modules/net-vpc-firewall/variables.tf [hunk 1]
```diff
@@ -14,44 +14,16 @@
...
-  description = "List of IP CIDR ranges for tag-based HTTP rule, defaults to the health checkers ranges."
-  type        = list(string)
-  default     = ["35.191.0.0/16", "130.211.0.0/22", "209.85.152.0/22", "209.85.204.0/22"]
-}
-
-variable "https_source_ranges" {
-  description = "List of IP CIDR ranges for tag-based HTTPS rule, defaults to the health checkers ranges."
-  type        = list(string)
-  default     = ["35.191.0.0/16", "130.211.0.0/22", "209.85.152.0/22", "209.85.204.0/22"]
+variable "admin_ranges_enabled" {
```
Comment (ludoo) [inline modules/net-vpc-firewall/variables.tf:23]: Can we remove this and use an empty list to prevent creation?

**Review by ludoo — APPROVED** [review]
we can maybe nuke the extra boolean var for admin ranges, WDYT?

File: modules/net-vpc-firewall/variables.tf (hunk 1 above)
Comment (juliocc) [reply modules/net-vpc-firewall/variables.tf:23]: We're already doing that

Comment (ludoo) [reply modules/net-vpc-firewall/variables.tf:23]: then why do we still have that variable? :)

**Comment by juliocc** [comment]
> we can maybe nuke the extra boolean var for admin ranges, WDYT?

Done

File: modules/net-vpc-firewall/variables.tf (hunk 1 above)
Comment (juliocc) [reply modules/net-vpc-firewall/variables.tf]: no idea, backwards compatibility ;)

**Pull request merged**